tests/functional/oauth_sign_up_verification_redirect.js

```diff
@@ -39,7 +36,7 @@ define([
           return self.remote.get(require.toUrl(url + '&verification_redirect=always'));
         })
         .then(function () {
-          return FunctionalHelpers.fillOutSignUp(self, email, PASSWORD, OLD_ENOUGH_YEAR);
+          return FunctionalHelpers.fillOutSignUp(self, email, PASSWORD);
```

Comment:
I haven't mentioned, but I am really glad you simplified the function signature.